src/process/functionality/image/list.js

```diff
@@ -0,0 +1,10 @@
+const Amplify = require('aws-amplify')
+const serializeAxiosResponse = require('../../../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../../../utils/defaultReqOpts')
+
```

Comment:
if we don't document what the q params are, we may want to reference to where someone can find them

Reply:
they should all be in the swagger docs which is linked in the top of the repo, or should we link directly to the endpoint docs for all functions ?